### ISSUE TITLE
Option to set log level

### ISSUE THREAD

**Issue by AnHeuermann** [ThummeTo/FMIImport.jl#50]
I'm using FMI.jl (and therefore FMIImport.jl) to simulate OpenModelica FMUs.
When running `fmu = FMI.fmiLoad(fmuPath)` I'm getting a bunch of info messages about empty files.

```julia
julia> fmu = FMI.fmiLoad(fmuPath)
[ Info: fmi2Unzip(...): Written file `binaries/linux64/simpleLoop_FMU.libs`, but file is empty.
[ Info: fmi2Unzip(...): Written file `sources/simpleLoop_11mix.h`, but file is empty.
[ Info: fmi2Unzip(...): Written file `sources/simpleLoop_includes.h`, but file is empty.
[ Info: fmi2Unzip(...): Written file `sources/simpleLoop_FMU.libs`, but file is empty.
[ Info: fmi2Unzip(...): Written file `sources/isfmi2`, but file is empty.
[ Info: fmi2Unzip(...): Successfully unzipped 229 files at `/tmp/fmijl_HTprAI/simpleLoop.interface`.
[ Info: fmi2Load(...): FMU resources location is `file:////tmp/fmijl_HTprAI/simpleLoop.interface/resources`
```

It would be nice to get the option to disable info prints from all and/or specific parts of FMImport.jl with some setting in FMI.jl.

My workaround for the moment is to use Suppressor.jl

```julia
local fmu
Suppressor.@suppress begin
  fmu = FMI.fmiLoad(fmuPath)
end
```

**Comment by ThummeTo** [comment]
Hi,

does this solve your issue: [https://thummeto.github.io/FMI.jl/dev/features/#Printing-internal-FMU-messages](https://thummeto.github.io/FMI.jl/dev/features/#Printing-internal-FMU-messages) ?

So `myFMU.executionConfig.logStatusOK=false` ?

Best regards!

**Comment by ThummeTo** [comment]
This will be part of FMI.jl v0.10.1 / FMIImport 0.12.0:

`function fmi2Load(...; logLevel::FMULogLevel=FMULogLevelWarn)` as default so no info messages are printed anymore, available options are `FMULogLevelInfo` (print the old info messages, `FMULogLevelWarn` print warnings and errors (default) and `FMULogLevelError` print errors only.

**Comment by AnHeuermann** [comment]
Sounds perfekt.

**Comment by ThummeTo** [comment]
Implemented in v0.12.0!
